app/controllers/application_controller.rb

```diff
@@ -1,13 +1,18 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
-  protect_from_forgery with: :exception
-
   before_action :configure_permitted_parameters, if: :devise_controller?
-  before_action :set_locale
 
   protected
 
+    def configure_permitted_parameters
```

Comment:
You have method duplication on line 8 and 16, but logic inside is a bit different

Reply:
fixed